### PR TITLE
Improve Notebooks/Replay documentation

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Updates documentation, getting ready for Notebooks updates and Replay service.
Should be merged as soon as the changes are approved in SSB/CHM

Adds
- RStudio
- CVMFS
- nb_conda_kernels
- Replay (instead of Binder)

---

**Related issue :**

### CODE REVIEW

**Review by sebastian-luna-valero — APPROVED** [review]
Thanks!

**Comment by gwarf** [comment]
@enolfc is this PR waiting as draft on something specific?

File: content/en/users/dev-env/replay/_index.md
Comment (sebastian-luna-valero) [inline content/en/users/dev-env/replay/_index.md:132]: Is the double `/hub/hub` correct?

```suggestion
[Replay JupyterHub home](https://replay.notebooks.egi.eu/hub/home):
```

**Review by sebastian-luna-valero — APPROVED** [review]
Minor comments below, thanks!

**Review by gwarf — APPROVED** [review]
LGTM, once all suggestions will have been addressed and the service will be in a good shape (certificate and so on)

File: content/en/users/dev-env/replay/_index.md
Comment (enolfc) [reply content/en/users/dev-env/replay/_index.md:132]: True!

**Review by gwarf — CHANGES_REQUESTED** [review]
The certificate for the service is not valid and should be replaced before this PR can be merged.

**Comment by enolfc** [comment]
> The certificate for the service is not valid and should be replaced before this PR can be merged.

replay hostname now in place. binder redirects to replay also.

File: content/en/users/dev-env/replay/_index.md
Comment (gwarf) [reply content/en/users/dev-env/replay/_index.md:101]: @enolfc, is this something you want to address before merging, or should we merge like this?

**Review by gwarf — APPROVED** [review]

File: content/en/users/dev-env/replay/_index.md
Comment (enolfc) [reply content/en/users/dev-env/replay/_index.md:101]: Image is now updated

**Pull request merged**
